ocaml/xapi/xapi_session.ml

```diff
@@ -661,6 +669,52 @@ let login_no_password_common ~__context ~uname ~originator ~host ~pool
   ignore (Client.Pool.get_all ~rpc ~session_id) ;
   session_id
 
+let login_no_password_common ~__context ~uname ~originator ~host ~pool
+    ~is_local_superuser ~subject ~auth_user_sid ~auth_user_name
+    ~rbac_permissions ~db_ref ~client_certificate =
+  Context.with_tracing ~originator ~__context __FUNCTION__ @@ fun __context ->
+  let internal_xapi_master_to_xapi_slave_login =
+    (xapi_internal_originator, true, true, None)
```

Comment:
I'd prefer to just put this tuple into the if-statement below, as on it's own these values are meaningless (what Booleans are these etc.).
```
if (originator, pool, is_local_superuser, uname) =
   (xapi_internal_originator, true, true, None)
```
make this clearer.